barretenberg/cpp/src/barretenberg/constants.hpp

```diff
@@ -11,6 +11,8 @@ static constexpr uint32_t CONST_PROOF_SIZE_LOG_N = 28;
 // circuits being folded.
 static constexpr uint32_t CONST_PG_LOG_N = 20;
 
+static constexpr uint32_t CONST_ECCVM_LOG_N = 16;
```

Comment:
Is this the right value? What kind of failure do we see if we exceed this? Might be nice to ensure that if its exceeded we get a straightforward error or print that lets us know that CONST_ECCVM_LOG_N is too small. We're sure to run into this very soon

Reply:
yeah, great point. I'll add some asserts/throw_or_aborts